build.sc

```diff
@@ -105,8 +107,8 @@ trait BasePublishModule extends BaseModule with CiReleaseModule {
   }
 }
 
-trait ScalaVersionModule extends ScalaModule with ScalafmtModule {
-  def scalaVersion = T.input("2.13.12")
+trait ScalaVersionModule extends CrossScalaModule with ScalafmtModule {
```

Comment:
I'm happy to see more modules being cross built, however I'd like to retain a `ScalaVersionModule` that is not cross built and that you can use in places where cross compilation is not required.

Reply:
Let's rename it to `Scala213Module` then